CMakeLists.txt

```diff
@@ -280,7 +281,7 @@ function (configure_test test_name)
 endfunction ()
 
 if (RUN_TEST)
-  configure_test (nfail_malloc_test)
+  #configure_test (nfail_malloc_test)
```

Comment:
this test is failing to build with boringssl

Reply:
Seems like a net win, nonetheless. After merging, I'll file a bug to fix this (or worst-case, add a cmake option for it.)